src/autocomplete.js

```diff
@@ -224,6 +225,10 @@ export default class Autocomplete extends Component {
     }
   }
 
+  handleInputClick (evt) {
+    this.handleInputChange(evt)
```

Comment:
I'm trying to go by how normal dropdowns work, and they don't open on focus. I think if you're tabbing through a page, having stuff open and close is probably not what you want.